app/Http/routes.php

```diff
@@ -180,11 +180,14 @@
         'show', 'edit', 'update'
     ]]);
 
-/*
-|-----------------------
-| Event Routes
-|-----------------------
-*/
+    Route::post('volunteer/{id}', array('uses' => 'VolunteerController@adminAssignValidator'));
+
+
+    /*
+    |-----------------------
+    | Event Routes
+    |-----------------------
+    */
     /**
```

Comment:
remove these changes. Add a documented part in routes.php for admins and validators